src/ds5/ds5-motion.cpp

```diff
@@ -269,7 +269,25 @@ namespace librealsense
 
         initialize_fisheye_sensor(ctx,group);
 
+        // D435i to use predefined values extrinsics
+        _depth_to_imu = std::make_shared<lazy<rs2_extrinsics>>([this]()
+        {
+            // BMI055 assembly transformation based on mechanical drawing (mm)
+            //    ([[ -1.  ,   0.  ,   0.  ,   5.52],
+            //      [  0.  ,   1.  ,   0.  ,   5.1 ],
+            //      [  0.  ,   0.  ,  -1.  , -11.74],
+            //      [  0.  ,   0.  ,   0.  ,   1.  ]])
+            // The orientation matrix will be integrated into the IMU stream data
+            pose ex = { {  1.f,     0.f,    0.f,
+                           0.f,     1.f,    0.f,
+                           0.f,     0.f,    1.f},
+                        { 0.00552f, -0.0051, -0.01174}};
+
+            return from_pose(ex);
+        });
+
         // Make sure all MM streams are positioned with the same extrinsics
+        environment::get_instance().get_extrinsics_graph().register_extrinsics(*_depth_stream, *_accel_stream, _depth_to_imu);
         environment::get_instance().get_extrinsics_graph().register_same_extrinsics(*_accel_stream, *_gyro_stream);
```

Comment:
While the translation doesn't matter for the gyro, I would prefer that the accel and gyro have the same transformation:
```suggestion
         environment::get_instance().get_extrinsics_graph().register_extrinsics(*_accel_stream, *_gyro_stream, _depth_to_imu);
```

Reply:
That's implemented in the original proposal.
Depth relates to a group of sensors with and explicit rigid body transform.
Within the group, all sensors are super positioned ( Identity transform):
Depth <--[Depth2IMU]-->   {Accel <--[ I ]--> Gyro}